cmd/bnsd/app/app_performance_test.go

```diff
@@ -41,7 +40,7 @@ func BenchmarkBnsdEmptyBlock(b *testing.B) {
 			return nil
 		})
 		if changed {
-			b.Fatal("unexpected change state")
+			b.Fatalf("unexpected change state")
```

Comment:
`go vet` will warn about `Xxxf` without arguments.

Reply:
Yet, your testing interface does not expose Fatal.
Can I update it to testing.TB?